Makefile

```diff
@@ -13,7 +13,7 @@ install:
 	go install ./...
 
 lint:
-	go vet ./...
+	cd ../../../; go tool vet -all -printfuncs=Verbosef,Infof,Debugf $$(go list github.com/windmilleng/tilt/... | grep -v /vendor/)
```

Comment:
is there a better way to do this? GOPATH isn't guaranteed to be set, and if I invoke `go list ./...` here I get:

```
go tool vet -all -printfuncs=Verbosef,Infof,Debugf $(go list ./... | grep -v /vendor/)
vet: error walking tree: stat github.com/windmilleng/tilt/cmd/tilt: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/build: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/cli: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/engine: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/image: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/k8s: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/logger: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/model: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/ospath: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/proto: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/service: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/testutils: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/tiltd: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/tiltd/tiltd_client: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/tiltd/tiltd_server: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/tiltfile: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/tracer: no such file or directory
vet: error walking tree: stat github.com/windmilleng/tilt/internal/watch: no such file or directory
vet: github.com/windmilleng/tilt/cmd/tilt: open github.com/windmilleng/tilt/cmd/tilt: no such file or directory
vet: no files checked
make: *** [lint] Error 1
```

Reply:
nvm I figured something out.